src/EprPrnIntegration.Api/UpdatePrnsFunction.cs

```diff
@@ -60,7 +61,19 @@ public async Task Run(
 
         // Send data to NPWD via pEPR API
         var npwdUpdatedPrns = PrnMapper.Map(updatedEprPrns, configuration);
-        var pEprApiResponse = await npwdClient.Patch(npwdUpdatedPrns, NpwdApiPath.UpdatePrns);
+        HttpResponseMessage? pEprApiResponse;
+        try
+        {
+            pEprApiResponse = await npwdClient.Patch(npwdUpdatedPrns, NpwdApiPath.UpdatePrns);
+        }
+        catch (Exception ex)
+        {
+
+            _emailService.SendEmailToNpwd(ex.Message);
+            logger.LogError(ex,  $"Failed to patch NpwdUpdatedPrns");            
+
+            pEprApiResponse = new HttpResponseMessage(System.Net.HttpStatusCode.NotFound);
```

Comment:
Can you be sure "Not Found" is the correct error?

Reply:
Why we are re-assigning response variable?